test-browser/tests/solidityUnittests.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+var init = require('../helpers/init')
+var sauce = require('./sauce')
+
+var sources = []
+
+module.exports = {
+  before: function (browser, done) {
+    init(browser, done)
+  },
+  '@sources': function () {
+    return sources
+  },
+  'Solidity Unittests': function (browser) {
+    runTests(browser)
+  },
+  tearDown: sauce
+}
+
+function runTests (browser) {
+  browser
+    .waitForElementVisible('#icon-panel', 10000)
+    .click('#icon-panel div[plugin="pluginManager"]')
+    .scrollAndClick('#pluginManager article[id="remixPluginManagerListItem_solidityUnitTesting"] button')
+    .clickLaunchIcon('fileExplorers')
```

Comment:
shouldn't you activate solidity plugin and compile a file?

Reply:
to run tests

Reply:
it is already activated when tests loads. see init.js

Reply:
and anyway remix-tests take care of compiling, so we only need the compile tab for getting the compiler version, not directly for compiling